src/frontend/src/flows/manage/recoveryMethodsSection.ts

```diff
@@ -26,19 +26,42 @@ export const recoveryMethodsSection = ({
   addRecoveryPhrase: () => void;
   addRecoveryKey: () => void;
 }): TemplateResult => {
+  const i18n = new I18n();
+
+  const copy = i18n.i18n(copyJson);
+  const wrapClasses = ["l-stack"];
+  const warnNoRecovery = isNullish(recoveryPhrase) && isNullish(recoveryKey);
+
+  if (warnNoRecovery) {
+    wrapClasses.push("c-card", "c-card--narrow", "c-card--warning");
```

Comment:
I don't like push as already explained. I know it's a bit opiniated but, also makes sense to some extend. So what you think of this?

```
const wrapClasses = ["l-stack", ...(warnNoRecovery ? ["c-card", "c-card--narrow", "c-card--warning"] : [])];
```

Reply:
I have no opinion on this. Happy to change it 👍

Reply:
Thanks